tasks/section5.yml

```diff
@@ -459,7 +609,7 @@
       - rule_5.2.15
 
 - name: "SCORED | 5.2.16 | AUDIT | Ensure SSH warning banner is configured"
-  command: /bin/true
+  command: grep "^Banner /etc/issue" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*